src/fabric-1.16.2/templates/plant.java.ftl

```diff
@@ -0,0 +1,218 @@
+<#-- @formatter:off -->
+
+<#include "mcitems.ftl">
+<#include "procedures.java.ftl">
+
+package ${package}.block;
+
+import net.fabricmc.loader.api.FabricLoader;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+public class ${name}FlowerBlock extends <#if data.plantType == "normal">Flower<#elseif data.plantType == "growapable">SugarCane</#if>Block <#if data.hasTileEntity>implements BlockEntityProvider</#if>{
+    public ${name}Block() {
+        super(FabricBlockSettings.of(Material.${data.material})
+        <#if data.unbreakable>
+                .strength(-1, 3600000)
+        <#else>
+                .strength(${data.hardness}F, ${data.resistance}F)
+        </#if>
+        .lightLevel(${(data.luminance * 15)?round})
+        <#if data.destroyTool != "Not specified">
+			    .breakByTool(FabricToolTags.${data.destroyTool?upper_case}S, ${data.breakHarvestLevel})
+        </#if>
+        .noCollision()
+        <#if data.slipperiness != 0.6>
+                .slipperiness(${data.slipperiness}F)
+        </#if>
+        .nonOpaque()
+		<#if data.emissiveRendering>
+                .emissiveLighting(${name}Block::always)
+        </#if>
```

Comment:
Do we still need this since it also overrides emissive rendering below
Also it will probably give a build error because ${name}Block won't exist

Reply:
didn't realize that, thank you for pointing it out.
now fixed